devfile/devfile-web
# chore(deps): updating vulnerable packages

## What does this PR do / why we need it
Updates packages with security issues.

## Which issue(s) does this PR fix

Fixes https://github.com/devfile/api/issues/1031

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
* `yarn install` to get the new packages
* `yarn audit` to check for vulnerabilities
